worlds/sims4/__init__.py

```
@@ -1,24 +1,25 @@
 # standard lib imports
-from typing import Mapping, Any, ClassVar
+from typing import Mapping, Any, ClassVar, Optional
```

Ruff (0.14.10)

Import from `collections.abc` instead: `Mapping`

Import from `collections.abc`

(UP035)